docs/src/main/asciidoc/central-logging.adoc

```diff
@@ -0,0 +1,369 @@
+
+First, you need to create a Fluentd image with the needed plugins: elasticsearch and input-gelf.
+You can use the following Dockerfile that should be created inside a `fluentd` directory.
+
+```
+FROM fluent/fluentd:v1.3-debian
+RUN ["gem", "install", "fluent-plugin-elasticsearch", "--version", "3.7.0"]
+RUN ["gem", "install", "fluent-plugin-input-gelf", "--version", "0.3.1"]
+```
+
+You can build the image or let docker-compose build it for you.
+
+Then you need to create a fluentd configuration file inside `$HOME/fluentd/fluent.conf`
+```
+<source>
+  type gelf
+  tag example.gelf
+  bind 0.0.0.0
+  port 12201
+</source>
+
+<match example.gelf>
+  @type elasticsearch
+  host elasticsearch
+  port 9200
+  logstash_format true
+</match>
+
+```
+
+Finally, launch the components that compose the EFK Stack:
+
+- Elasticsearch
+- Fluentd
+- Kibana
+
+You can do this via the following docker-compose file that you can launch via `docker-compose run -d`:
+```
+version: '3.2'
+
+services:
+  elasticsearch:
+    image: docker.elastic.co/elasticsearch/elasticsearch-oss:6.8.2
+    ports:
+      - "9200:9200"
+      - "9300:9300"
+    environment:
+      ES_JAVA_OPTS: "-Xms512m -Xmx512m"
+    networks:
+      - efk
+
+  fluentd:
+    build: fluentd
+    ports:
+      - "12201:12201/udp"
+    volumes:
+      - source: $HOME/fluentd
+        target: /fluentd/etc
+        type: bind
+    networks:
+      - efk
+    depends_on:
+      - elasticsearch
+
+  kibana:
+    image: docker.elastic.co/kibana/kibana-oss:6.8.2
+    ports:
+      - "5601:5601"
+    networks:
+      - efk
+    depends_on:
+      - elasticsearch
+
+networks:
+  efk:
+    driver: bridge
+```
+
+Launch your application, you should see your logs arriving inside EFK: you can use Kibana to access them.
+
+== Fluentd alternative: use SYSLOG
+
+You can also send your logs to Fluentd using a SYSLOG input.
+As opposed to GELF input, SYSLOG input will not render multiline logs in one event, that's why we advise to use the GELF input that we implement in Quarkus.
+
+First, you need to create a Fluentd image with the needed plugins: elasticsearch and input-gelf.
+You can use the following Dockerfile that should be created inside a `fluentd` directory.
+
+```
+FROM fluent/fluentd:v1.3-debian
+RUN ["gem", "install", "fluent-plugin-elasticsearch", "--version", "3.7.0"]
+```
+
+Then, you need to create a fluentd configuration file inside `$HOME/fluentd/fluent.conf`
+```
+<source>
+  @type syslog
+  port 5140
+  bind 0.0.0.0
+  message_format rfc5424
+  tag system
+</source>
+
+<match **>
+  @type elasticsearch
+  host elasticsearch
+  port 9200
+  logstash_format true
+</match>
+```
+
+Then, launch the components that compose the EFK Stack:
+
+- Elasticsearch
+- Fluentd
+- Kibana
+
+You can do this via the following docker-compose file that you can launch via `docker-compose run -d`:
+```
+version: '3.2'
+
+services:
+  elasticsearch:
+    image: docker.elastic.co/elasticsearch/elasticsearch-oss:6.8.2
```

Comment:
Same here about Elastic version.

Maybe it's worth making it a variable of the Asciidoc file?